packages/lib-classifier/src/components/Classifier/components/ModalTutorial/ModalTutorial.js

```diff
@@ -23,21 +23,26 @@ function storeMapper (stores) {
 @inject(storeMapper)
 @observer
 class ModalTutorial extends React.Component {
+  onClose () {
+    const { setModalVisibility } = this.props
+    setModalVisibility(false)
+  }
+
   render () {
-    const { loadingState, showModal, setModalVisibility, tutorial } = this.props
+    const { loadingState, showModal, tutorial } = this.props
     if (loadingState === asyncStates.success && tutorial) {
       return (
         <Modal
           {...this.props}
           active={showModal}
-          closeFn={() => { setModalVisibility(false) }}
+          closeFn={this.onClose.bind(this)}
           title={counterpart('ModalTutorial.title')}
         >
           <ResponsiveContext.Consumer>
             {size => {
               const width = (size === 'small') ? '100%' : '330px'
               return (
-                <SlideTutorial pad='none' width={width} />
+                <SlideTutorial onClick={this.onClose.bind(this)} pad='none' width={width} />
```

Comment:
Styling: you sometimes list props, sometimes write them inline like here. Should they always be listed vertically?

Reply:
I tend to list them vertically when there are 3+ / some are objects